WordPress/openverse
# Change outdated links in about page for codebase and project board to current links

## Fixes



Fixes #3757 by @obulat

## Description


This PR changes the outdated links in the about page to the current codebase and project board, "All of our code is open source and can be accessed at the [Openverse GitHub repository](https://github.com/wordpress/openverse). We [welcome community contribution](https://docs.openverse.org/general/contributing.html). You can see what [we’re currently working on](https://github.com/orgs/WordPress/projects/75)."

<img src="https://github.com/WordPress/openverse/assets/66654126/86d09ffb-5f42-4a52-9c9e-6bbc25e83474" width="509">


## Testing Instructions

Run ```just frontend/run dev```, verify that the about page has the updated links.

## Checklist


- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
